com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs

```diff
@@ -88,6 +95,10 @@ static CoreEditorStyles()
             contextMenuIcon = new GUIContent(paneOptionsIcon, contextTooltip);
             contextMenuStyle = new GUIStyle("IconButton");
 
+            redTexture = CoreEditorUtils.CreateColoredTexture2D(Color.red);
+            greenTexture = CoreEditorUtils.CreateColoredTexture2D(Color.green);
+            blueTexture = CoreEditorUtils.CreateColoredTexture2D(Color.blue);
```

Comment:
Not sure it will be used outside of HDRP though

Reply:
I will leave them here for the future.